packages/taiko-client/cmd/flags/proposer.go

```diff
@@ -95,6 +97,14 @@ var (
 		Category: proposerCategory,
 		EnvVars:  []string{"L1_REVERT_PROTECTION"},
 	}
+	// Preconfirmation related
+	FallbackTimeout = &cli.DurationFlag{
+		Name:     "preconfirmation.fallback.timeout",
+		Usage:    "Timeout to propose L2 transactions as a fallback preconfer",
+		Category: proposerCategory,
+		Value:    5 * time.Minute,
```

Comment:
```suggestion
		Value:    2 * time.Minute,
```

Let's use 2min at first.